### PR TITLE
Asset transfer : Cloudinary -> TinaCloud 

### PR DESCRIPTION
**Pull request by joshbermanssw**
This PR is in address [🚒 Cloudinary -> TinaCloud for asset management](https://github.com/tinacms/tina.io/issues/2344)

We are moving our media manager from Cloudinary to Tina Cloud.

As per the instructions in https://github.com/tinacms/tinacloud/issues/2348#issuecomment-2415901826 i have used the script @ncn-ssw set-up and manually one by one transferred all of this media into our repo to be used with Tina Cloud. I would check references to each asset and improve the file structure for when i bring media into our local.

We have also moved away from .gif and have instead chosen to use .webm 

As a result we were not able to render our .webm media via markdown []() syntax so i create a WebmEmbed using `<video>` to display the content. This has been used where appropriate. 

FInally i have changed the config to point towards Tina Cloud rather than Cloudinary

### CODE REVIEW

File: content/blog/TinaCMS-New-Public-Roadmap.mdx [hunk 1]
```diff
@@ -7,14 +7,14 @@ author: James O'Halloran
...
 
-![Public Roadmap](https://res.cloudinary.com/forestry-demo/image/upload/v1669232450/blog-media/roadmap/Screen_Shot_2022-11-23_at_2.22.24_PM.png)
+![Public Roadmap](/img/blog/tinacms-new-public-roadmap/public-roadmap.png)
 
 The roadmap is reserved for items in which we have a high degree of confidence that they'll fall into a given timeframe. Each roadmap item will link to a GitHub discussion, where you can comment on features, or subscribe for updates.\
 \
 If you'd like to see something make its way into the roadmap, start by creating a feature request in our [GitHub Discussions](https://github.com/tinacms/tinacms/discussions "GitHub Discussions").
 
 Under the hood, the roadmap is also managed with Tina! So we'll be jumping it and keeping this page up to date regularly as plans change and as progress is made. \
 \
```
Comment (ncn-ssw) [inline content/blog/TinaCMS-New-Public-Roadmap.mdx:17]: **issue**: Though this line was not changed, the edit to this file now means that the slash is being displayed.

image
**Figure: Slash added to blog post**

File: content/blog/from-cms-to-contextual.mdx [hunk 2]
```diff
@@ -6,7 +6,7 @@ author: James Perkins
 
 Tina allows you as a developer to create an amazing editing experience. By default the editing experience is a more traditional CMS, where you login to a specific URL and you edit your content without being able to see the content till after you finish your changes. But, if you are looking for a more transparent real-time editing experience, Tina has a superpower, called Visual Editing. Just as it sounds, you get instant feedback on the page as well as being able to preview the changes before publishing live to your site.
 
-![Example of Tina Context editing](https://res.cloudinary.com/forestry-demo/video/upload/q_100/v1647436971/blog-media/cms-to-contextual/TinaSuper.gif)
+<WebmEmbed embedSrc="/img/blog/from-cms-to-contextual/TinaSuper.webm"/>
```
Comment (ncn-ssw) [inline content/blog/from-cms-to-contextual.mdx:9]: **nitpick**: Are we able to get a better WebM conversion for this animation?

The lack of anti-aliasing is obvious and makes the animation look less professional.

image
**Figure: The WebM conversion on the left, the old GIF on the right, with the screenshot taken at 100% magnification both windows - note the ragged edges on the title text**

File: content/blog/mdx-powered-docs.mdx
Comment (ncn-ssw) [inline content/blog/mdx-powered-docs.mdx:1]: **issue**: Numerous images on this page have not been transferred.

image
**Figure: Missing images**

File: content/blog/simple-markdown-blog-nextjs.mdx
Comment (ncn-ssw) [inline content/blog/simple-markdown-blog-nextjs.mdx:1]: **issue**: Animation not transferred.

image
**Figure: The old site is on the right, with the image that was not transferred surrounded by a red box. An arrow indicates where it should be on the updated site.**

File: content/blog/tinasaurus-docusaurus-starter.mdx [hunk 3]
```diff
@@ -7,7 +7,7 @@ author: Scott Gallant & Scott Byrne
 
 We're excited to share [Tinasaurus](https://github.com/tinacms/tinasaurus ""), a Docusaurus starter site with support for TinaCMS.
 
-<WideImage src="https://res.cloudinary.com/forestry-demo/image/upload/v1677522334/tina-io/blog/tinacms-docusaurus-markdown-editing.gif" alt="Editing Docusaurus content with a CMS" />
+<Youtube embedSrc="/img/blog/tinasaurus-docusaurus-starter/tinacms-docusaurus-markdown-editing.webm"/>
```
Comment (ncn-ssw) [inline content/blog/tinasaurus-docusaurus-starter.mdx:10]: **question**: Is this the correct replacement component, given that the movie does not autoplay?

https://tina.io/blog/tinasaurus-docusaurus-starter/

image
**Figure: For the updated site on the left, the movie is poorly proportioned with plenty of black above and below. The movie also does not autoplay.**

File: content/docs/contextual-editing/react.mdx
Comment (ncn-ssw) [inline content/docs/contextual-editing/react.mdx:1]: **Issue**: Animation not transferred across.

https://tina.io/docs/contextual-editing/react/

image
**Figure: The raw WebmEmbed component text on the updated website on the left**

File: content/docs/contextual-editing/router.mdx [hunk 4]
```diff
@@ -74,8 +74,6 @@ const default defineConfig({
 
 Now when a document is clicked in the CMS we will be re-directed to the page in the site with visual editing.
 
-<CloudinaryVideo src="https://res.cloudinary.com/forestry-demo/video/upload/q_100,h_584/e_accelerate:-20/v1655919318/tina-io/docs/RoutMapperVid" />
```
Comment (ncn-ssw) [inline content/docs/contextual-editing/router.mdx:77]: **question**: Why was this video not transferred?

File: content/docs/contextual-editing/tinafield.mdx [hunk 5]
```diff
@@ -7,7 +7,7 @@ title: The tinafield helper
 
 Tina's "click to edit" feature allows editors to select the element they want to edit on the page in order to see it in the sidebar.
 
-<CloudinaryVideo src="https://res.cloudinary.com/forestry-demo/video/upload/q_80,h_584/v1684344056/tina-io/docs/quick-edit-demo" />
+<WebmEmbed embedSrc="/video/quick-edit-demo.mp4"/>
```
Comment (ncn-ssw) [inline content/docs/contextual-editing/tinafield.mdx:10]: **Issue**: WebmEmbed not being rendered correctly.

image
**Figure: Updated site on the left**

File: content/blog/TinaCMS-New-Public-Roadmap.mdx (hunk 1 above)
Comment (joshbermanssw) [reply content/blog/TinaCMS-New-Public-Roadmap.mdx]: Fixed

File: content/blog/mdx-powered-docs.mdx
Comment (joshbermanssw) [reply content/blog/mdx-powered-docs.mdx:1]: Hmm this is weird - it was rendering on my local. 

I found that the images were being served from https://res.craft.do/user/full/c67cad1b-6dc6-4909-0f8e-19d468ba9fd4/doc/F38A8651-6C07-422F-A3E4-DBB7464B4570/<ID>

So ive just moved all the images into the repo and called them internally - images render on my local still

File: content/blog/tinasaurus-docusaurus-starter.mdx (hunk 3 above)
Comment (joshbermanssw) [reply content/blog/tinasaurus-docusaurus-starter.mdx]: Fixed - changed to `<WebmEmbed/>`

File: content/docs/contextual-editing/router.mdx (hunk 4 above)
Comment (joshbermanssw) [reply content/docs/contextual-editing/router.mdx:77]: I cannot get access to this file

Comment (ncn-ssw) [reply content/docs/contextual-editing/router.mdx:77]: You can access the file through https://tina.io/docs/contextual-editing/router/

File: content/blog/simple-markdown-blog-nextjs.mdx
Comment (joshbermanssw) [reply content/blog/simple-markdown-blog-nextjs.mdx:1]: Resolved - added back

File: content/docs/contextual-editing/tinafield.mdx (hunk 5 above)
Comment (joshbermanssw) [reply content/docs/contextual-editing/tinafield.mdx:10]: Resolved

File: content/blog/from-cms-to-contextual.mdx (hunk 2 above)
Comment (joshbermanssw) [reply content/blog/from-cms-to-contextual.mdx:9]: Reviewed with Landon - the quality of the .webm matches the quality of the .gif but at 1/10 of the size

Comment (joshbermanssw) [reply content/blog/from-cms-to-contextual.mdx:9]: Spoke with @landonmaxwell - its a render issue with the scaling on its render. Although the component `WebmEmbed` doesnt directly dictate anything to do with scaling.

File: content/docs/contextual-editing/router.mdx (hunk 4 above)
Comment (joshbermanssw) [reply content/docs/contextual-editing/router.mdx:77]: as per our conversation on teams - this is resolved

File: content/docs/editing/overview.mdx [hunk 6]
```diff
@@ -19,10 +19,11 @@ date: '2021-01-01T00:00:00.000Z'
 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultricies urna ut ex varius, sed fringilla nibh posuere. Vestibulum a pulvinar eros, vel varius orci. Sed convallis purus sed tellus pellentesque ornare quis non velit. Quisque eget nibh nec nisl volutpat aliquet. Donec pharetra turpis vitae diam aliquam rutrum. Sed porta elit ut mi vehicula suscipit. Ut in pulvinar nunc.
 ```
 
-![markdown-editing](https://res.cloudinary.com/forestry-demo/image/upload/v1645712826/tina-io/docs/markdown.gif)
+<WebmEmbed embedSrc="/img/docs/editing/markdown.webm"/>
```
Comment (ncn-ssw) [inline content/docs/editing/overview.mdx:22]: **issue**: The spacing around this image needs to be corrected, as it is colliding with the text block above it.

image
**Figure: There is no spacing above the updated image on the left, where there was previously**

File: content/docs/editing/single-document-collections.mdx [hunk 7]
```diff
@@ -38,8 +38,6 @@ export default defineConfig({
 
 Once both settings have been applied, and the document contains a single file, the UI will forego the list page and direct the user straight to the document.
 
-<CloudinaryVideo src="https://res.cloudinary.com/forestry-demo/video/upload/q_100,h_584/e_accelerate:-20/tina-io/docs/extending-tina/Single_Collection_Demo" />
-
```
Comment (ncn-ssw) [inline content/docs/editing/single-document-collections.mdx:42]: **question**: Why was this video not transferred?

File: content/docs/editing/overview.mdx (hunk 6 above)
Comment (joshbermanssw) [reply content/docs/editing/overview.mdx:24]: Resolved - i also added some context for the video

File: content/docs/product-tour.mdx [hunk 8]
```diff
@@ -14,94 +14,94 @@ Here we give a quick **2 minute overview of TinaCMS**, and how it fits into your
 
 TinaCMS is an open-source Content Management System (CMS) that seamlessly integrates with your Markdown workflow.
 
-![Intro-gif](https://res.cloudinary.com/forestry-demo/image/upload/v1645712511/tina-io/docs/your-blocks.gif)
+<WebmEmbed embedSrc="/img/docs/your-blocks.webm"/>
```
Comment (ncn-ssw) [inline content/docs/product-tour.mdx:17]: **question**: There is an image being shown where there wasn't 1 previously - is this intentional?

image
**Figure: New image on the updated page at left, which also appears to be shown in the gutter image**

File: content/docs/product-tour.mdx [hunk 9]
```diff
@@ -14,94 +14,94 @@ Here we give a quick **2 minute overview of TinaCMS**, and how it fits into your
...
 
-![Queryable-db-png](https://res.cloudinary.com/forestry-demo/image/upload/v1689874024/tina-io/docs/product-tour/queryable_database.png)
+![Queryable-db-png](/img/docs/product-tour/queryable_database.png)
 
 ### Git-backed Content
 
 - All your content gets backed by git into JSON/Markdown/MDX files
 
-![Git-backed-png](https://res.cloudinary.com/forestry-demo/image/upload/v1689874025/tina-io/docs/product-tour/git-backed.png)
+![Git-backed-png](public/img/docs/product-tour/git-backed.png)
```
Comment (ncn-ssw) [inline content/docs/product-tour.mdx:41]: **issue**: Image not loaded due to "public" filename prefix.

File: content/docs/reference/rich-text-usage/markdown-tables.mdx [hunk 10]
```diff
@@ -46,7 +46,7 @@ const MyTable = props => <table>
 
 ## Showcase
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1726549733/tina-io/docs/2024-09-13_15-59-11_1_y2elbn.gif)
+<WebmEmbed embedSrc="public/img/docs/reference/cells-and-rows.webm"/>
```
Comment (ncn-ssw) [inline content/docs/reference/rich-text-usage/markdown-tables.mdx:49]: **issue**: Images not shown due to 'public' prefix.

File: content/docs/reference/types/object.mdx [hunk 11]
```diff
@@ -45,15 +45,15 @@ Ultimately there are 3 ways to configure this type...
 
 * With `list` set to `false`, the `fields` array can be used to wrap some data – this appears in the editor as a nested page. This can be used to organise the CMS, such as grouping options together, as in the gif below.
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331124/tina-io/docs/Sep-26-2024_16-05-37_d2ol8w.gif)
+<WebmEmbed embedSrc="public/img/docs/reference/rodmap-grid.webm"/>
```
Comment (ncn-ssw) [inline content/docs/reference/types/object.mdx:48]: **issue**: Image not shown due to 'public' prefix.

File: content/docs/reference/types/object.mdx [hunk 12]
```diff
@@ -45,15 +45,15 @@ Ultimately there are 3 ways to configure this type...
...
 
 * With `list` set to `true`, the `fields` array contains the data structure to be repeated – this appears in the editor as above, but grouped as a list.
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331115/tina-io/docs/Sep-26-2024_16-03-00_f3ogev.gif)
+<WebmEmbed embedSrc="public/img/docs/reference/roadmap-items.webm"/>
```
Comment (ncn-ssw) [inline content/docs/reference/types/object.mdx:52]: **issue**: Image not shown due to 'public' prefix.

File: content/docs/reference/types/object.mdx [hunk 13]
```diff
@@ -45,15 +45,15 @@ Ultimately there are 3 ways to configure this type...
...
 
 * With `list` set to `true`, and using `templates` as opposed to `fields` the user can select between associated templates – this appears as above, but pressing the `+`  gives you the option to choose between templates.
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331134/tina-io/docs/Sep-26-2024_16-07-02_zv5uth.gif)
+<WebmEmbed embedSrc="public/img/docs/reference/text-and-media-component.webm"/>
```
Comment (ncn-ssw) [inline content/docs/reference/types/object.mdx:56]: **issue**: Image not shown due to 'public' prefix.

File: content/docs/reference/types/rich-text.mdx
Comment (ncn-ssw) [inline content/docs/reference/types/rich-text.mdx:1]: **issue**: Images not transferred.

image
**Figure: On the updated page at left, the images cannot load**

File: tina/tina-lock.json
Comment (ncn-ssw) [inline tina/tina-lock.json:1]: **issue**: The file `tina/collectionsSchema/examples.tsx` is still using a reference to Cloudinary files, and that is causing a Cloudinary reference to make its way into this `tina-lock.json` file.

File: content/docs/product-tour.mdx (hunk 9 above)
Comment (joshbermanssw) [reply content/docs/product-tour.mdx:41]: Resolved

File: content/docs/reference/rich-text-usage/markdown-tables.mdx (hunk 10 above)
Comment (joshbermanssw) [reply content/docs/reference/rich-text-usage/markdown-tables.mdx:49]: Resolved

File: content/docs/reference/types/object.mdx (hunk 11 above)
Comment (joshbermanssw) [reply content/docs/reference/types/object.mdx:48]: Done resolved

File: content/docs/reference/types/object.mdx (hunk 12 above)
Comment (joshbermanssw) [reply content/docs/reference/types/object.mdx:52]: Resolved

File: content/docs/reference/types/object.mdx (hunk 13 above)
Comment (joshbermanssw) [reply content/docs/reference/types/object.mdx:56]: Resolved

File: content/docs/product-tour.mdx (hunk 8 above)
Comment (joshbermanssw) [reply content/docs/product-tour.mdx]: Resolved

File: content/docs/reference/types/rich-text.mdx
Comment (joshbermanssw) [reply content/docs/reference/types/rich-text.mdx:1]: Fixed

File: tina/tina-lock.json
Comment (joshbermanssw) [reply tina/tina-lock.json:1]: resolved

File: content/docs/editing/single-document-collections.mdx [hunk 14]
```diff
@@ -38,6 +38,8 @@ export default defineConfig({
 
 Once both settings have been applied, and the document contains a single file, the UI will forego the list page and direct the user straight to the document.
 
+<WebmEmbed embedSrc="/video/Single Collection Demo.webm"/>
```
Comment (ncn-ssw) [inline content/docs/editing/single-document-collections.mdx:41]: **issue**: Avoid spaces in names - https://www.ssw.com.au/rules/remove-spaces-from-your-folders-and-filename/

File: content/docs/reference/types/object.mdx [hunk 15]
```diff
@@ -45,15 +45,15 @@ Ultimately there are 3 ways to configure this type...
 
 * With `list` set to `false`, the `fields` array can be used to wrap some data – this appears in the editor as a nested page. This can be used to organise the CMS, such as grouping options together, as in the gif below.
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1727331124/tina-io/docs/Sep-26-2024_16-05-37_d2ol8w.gif)
+<WebmEmbed embedSrc="/img/docs/reference/rodmap-grid.webm"/>
```
Comment (ncn-ssw) [inline content/docs/reference/types/object.mdx:48]: **issue**: The quality of this image and those below it is very poor - they either need to be re-encoded or shrunk.

image
**Figure: The image is pixelated so badly that the point of the animation is lost**

File: content/docs/editing/single-document-collections.mdx (hunk 14 above)
Comment (joshbermanssw) [reply content/docs/editing/single-document-collections.mdx]: Resolved

File: content/docs/reference/types/object.mdx (hunk 15 above)
Comment (joshbermanssw) [reply content/docs/reference/types/object.mdx]: We have added the option for users to specify the width. for these webm's we use 50% width so we shrink them into a better resolution

**Review by ncn-ssw — APPROVED** [review]
🦙 Approved

**Pull request merged**